src/server/routes/plywood/plywood.ts

```diff
@@ -49,9 +77,9 @@ export function plywoodRouter(getSettings: SettingsGetter) {
       }
     }
 
-    let ex: Expression = null;
+    let parsedExpression: Expression = null;
```

Comment:
Encapsulate parsing somewhere (perhaps together with applySubset / filterMain)

Reply:
But parsing here is simple, complex logic is around catching error and sending appropriate response which I think is role of route.